backend/src/org/commcare/xml/SuiteParser.java

```diff
@@ -33,6 +33,8 @@
     ResourceTable table;
     String resourceGuid;
 
+    private boolean skipResources = false;
```

Comment:
Hm, plausible. The app fixtures are tiny right now, so I'm inclined to keep doing it until I have a chance to consider it more deeply